AssemblyUtil.spec

```diff
@@ -52,6 +52,12 @@ module AssemblyUtil {
             workspace_name - target workspace
             assembly_name - target object name
 
+            type - should be one of isolate', 'metagenome', (maybe 'transcriptome')
+
+            min_contig_length - if set and value is greater than 1, this will only include sequences
```

Comment:
I'd say I left it in just to test you, but I can't actually see a typo on that line :)  Oh wait- there is a missing quote above on line 55 in front of isolate- if that's what you found I'm impressed!